src/atoms/Checkbox.tsx

```diff
@@ -11,7 +11,7 @@ const Checkbox: FC<Props> = ({
   className,
   name,
   defaultChecked = false,
-  onChange = (event) => {},
+  onChange = () => undefined,
```

Comment:
no-empty-function

Reply:
Is `undefined` better than `void`?

```suggestion
  onChange = () => void,
```

Reply:
`void` cannot use here. We need to put an actual value here rather than a type declaration.

Reply:
Sure!

📝